whois-api/src/main/java/net/ripe/db/whois/api/elasticsearch/AccountingElasticSearchCallback.java

```diff
@@ -0,0 +1,64 @@
+package net.ripe.db.whois.api.elasticsearch;
+
+import com.google.common.net.InetAddresses;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import net.ripe.db.whois.common.source.Source;
+import net.ripe.db.whois.query.QueryMessages;
+import net.ripe.db.whois.query.acl.AccessControlListManager;
+import net.ripe.db.whois.query.domain.QueryCompletionInfo;
+import net.ripe.db.whois.query.domain.QueryException;
+
+import java.io.IOException;
+import java.net.InetAddress;
+
+public abstract class AccountingElasticSearchCallback<T> {
```

Comment:
I suggest you re-order the words as ```ElasticSearchAccountingCallback``` as it's more readable (Or ```ElasticAccountingCallback``` to be consistent with the other class names)